spring-cloud-dataflow-docs/src/main/asciidoc/configuration-kubernetes-app-properties.adoc

```diff
@@ -1134,6 +1134,11 @@ The following example shows how you can configure an Init Container for an appli
 [source,options=nowrap]
 ----
 deployer.<application>.kubernetes.initContainer={containerName: 'test', imageName: 'busybox:latest', commands: ['sh', '-c', 'echo hello']}
+# alternative for multiple init containers
+deployer.<application>.kubernetes.initContainers=[{containerName:'test',imageName:'busybox:latest',commands:['sh','-c','echo hello']},{containerName:'test2',imageName:'busybox:latest',commands:['sh','-c','echo world']}]
```

Comment:
Nitpick :  Make sure we have a space after the comma so samples line up with each other.

Reply:
Should the alternative example go below the multiple containers example?

Reply:
inidividually is misspelled ;-)

Reply:
Fixed